### ISSUE TITLE
Angular Dockerfile optimization

### ISSUE THREAD

**Issue by ghost** [Elwafi-Elmehdi/stuff-as-code#1]
I know it's a very small one but I think it's better to copy the package*.json files then after installing the dependencies we can copy everything. In this case we can decrease our image build times because only if we change dependencies then only it runs the npm install

**Comment by JohnsonBervigwm** [comment]
Hey Akashnani, I totally agree with your idea! Even small optimizations like this can make a big difference in our workflow and save us a lot of time. It's always great to find ways to speed up our development process, and your suggestion is a great example of that.

**Comment by ghost** [comment]
Yes that's right Johnson, let me create a PR for this issue and let's resolve.
